examples/bigquery_user_info_updater_tool/README.md

```diff
@@ -0,0 +1,317 @@
+row will contain values and all other columns will be null. If you want to query only
+ the most up to date, full record, the solution could easily become complicated and inefficient.
+
+In the case mentioned above, imagine we have a destination table called
+`user_info_updates`,  which is populated by a Dataflow pipeline that gathers and
+ingests data from users registering or updating their profile on a website. 
+When users register for the first time, an entire row is ingested in the table:
+##### user_info_updates 
+
+ | user_id      | ingestTimestamp    | attribute1     |attribute2    |attribute3     |
+| :------------- | :----------- | :----------- | :----------- |  :----------- | 
+|  userA | 1   | xyz  |abc  |qwerty|
+|  userB | 2   | poc  |mvp  |ytrewq|
+|  userC | 3   | zyx  |pvm  |qywter|
+
+Now imagine that userA and userB update their information. In this case the 
+data source will send only a partial updated row per update: 
+
+
+##### user_info_updates 
+
+ user_id      | ingestTimestamp    | attribute1     |attribute2    |attribute3     |
+| :------------- | :----------- | :----------- | :----------- |  :----------- | 
+|  userA | 1   | xyz  |abc  |qwerty|
+|  userB | 2   | poc  |mvp  |ytrewq|
+|  userC | 3   | zyx  |pvm  |qywter|
+|  userA | 4   |   |ghj  | |
+|  userA | 5   |   |  |lot|
+|  userB | 6   |  cor |  | |
+|  userA | 7   |   | dat ||
+
+These partial rows are not very useful since any non-updated values are missing. 
+To provide analysts/data scientists/etc with the latest, full record for each user,
+ a table called user_info_final can be created using a series of SQL queries: 
+
+##### user_info_final
+
+ user_id      | ingestTimestamp    | attribute1     |attribute2    |attribute3     |
+| :------------- | :----------- | :----------- | :----------- |  :----------- | 
+|  userA | 7   | xyz  |dat  |lot|
+|  userB | 6   | cor  |mvp  |ytrewq|
+|  userC | 3   | zyx  |pvm  |qywter|
+
+The BigQuery User Info Updater to can be used to solve this problem with series of advanced
+queries, creating a table with one up-to-date row per customer. 
+
+## Tables 
+
+Before the SQL queries can be written, three tables need to be created:  `user_info_updates`, 
+`user_info_updates_temp`, and `user_info_final`. 
+
+All new and updated rows will be stored in `user_info_updates`, so it will serve 
+as a historical table. `user_info_updates_temp` will hold the truncated results 
+of a query described below. `user_info_final` will hold one row for each customer, 
+and each row will hold the most up to date data relative to the last update. 
+
+All tables will share the same schema. While the particular schema does not matter, 
+it does need to include a unique ID for each user and a timestamp marking when 
+the row was ingested into BigQuery. Throughout this document, these fields will
+ be called `userId` and `ingestionTimestamp`.
+
+Note that the table names and fields listed here can be changed to anything you 
```

Comment:
jaketf on Aug 27, 2019 Member
Should we call this asset something different then?
Perhaps BigQuery Partial Update Aggregation

 @annarudy
annarudy on Oct 4, 2019 Author Member
I do like that name. Let me run that by Lorenzo(the other author).

Reply:
it looks like the asset itself was already published, so we probably shouldn't change the name